inc/upgrade.php

```diff
@@ -145,12 +148,7 @@ function responsive_upgrade_091( $verbose = true ) {
 		)
 	);
 
-	$banner_query = sprintf(
-		'SELECT post_id, meta_value FROM %s WHERE meta_key = "_bu_banner"',
-		$wpdb->postmeta
-	);
-
-	$results = $wpdb->get_results( $banner_query );
+	$results = $wpdb->get_results( 'SELECT post_id, meta_value FROM $wpdb->postmeta WHERE meta_key = "_bu_banner"' );
```

Comment:
@philcable The Unit tests are failing. It appears to be around this line and because the unit tests.

```FFWordPress database error You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '->postmeta WHERE meta_key = "_bu_banner"' at line 1 for query SELECT post_id, meta_value FROM $wpdb->postmeta WHERE meta_key = "_bu_banner" made by PHPUnit_TextUI_Command::main, PHPUnit_TextUI_Command->run, PHPUnit_TextUI_TestRunner->doRun, PHPUnit_Framework_TestSuite->run, PHPUnit_Framework_TestSuite->run, PHPUnit_Framework_TestCase->run, PHPUnit_Framework_TestResult->run, PHPUnit_Framework_TestCase->runBare, PHPUnit_Framework_TestCase->runTest, ReflectionMethod->invokeArgs, Tests_Responsive_Framework_Upgrades->test_responsive_framework_upgrade, responsive_framework_upgrade, responsive_upgrade_091
739FWordPress database error You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '->postmeta WHERE meta_key = "_bu_banner"' at line 1 for query SELECT post_id, meta_value FROM $wpdb->postmeta WHERE meta_key = "_bu_banner" made by PHPUnit_TextUI_Command::main, PHPUnit_TextUI_Command->run, PHPUnit_TextUI_TestRunner->doRun, PHPUnit_Framework_TestSuite->run, PHPUnit_Framework_TestSuite->run, PHPUnit_Framework_TestCase->run, PHPUnit_Framework_TestResult->run, PHPUnit_Framework_TestCase->runBare, PHPUnit_Framework_TestCase->runTest, ReflectionMethod->invokeArgs, Tests_Responsive_Framework_Upgrades->test_responsive_upgrade_091, responsive_upgrade_091
740FFFFFF...         118 / 118 (100%)```

Reply:
@stonybrookweb @philcable Oops, thanks for catching that!

I just addressed this in https://github.com/bu-ist/responsive-framework/pull/197/commits/95ddd4a750c9133584b612df718cf9acf7bc9a93 by using `"` around the queries with interpolated `$wpdb` data. I'll do a quick check across the other projects to make sure that didn't pop up elsewhere.

https://github.com/bu-ist/responsive-framework/pull/197/commits/8ddb4e517489ca1d9306c9ac4d1de8b082579bf5 then removes a piece from the unit tests that was ignoring incorrect use of `$wpdb->prepare()` and the tests are passing.

I left the conflict with the changelog as is for now.